addons/knobs/src/angular/knob-wrapper.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+
+@Component({
+  selector: 'knobs-component',
+  template: `
+    <div>
```

Comment:
An empty div? Is this uncompleted / in progress or is this the way it should be?